payment/urls.py

```diff
@@ -4,6 +4,7 @@
 urlpatterns = [
     path('list', views.payment_list, name="payment_list"),
     path('create', views.payment_create, name="payment_create"),
+    path('delete/<int:payment_id>', views.payment_delete, name="payment_delete")
```

Comment:
I have been seeing that some team members are forming the urls of the form **_<int:payment_id>/delete_**.Perhaps we could define how they are going to be defined so that they are created in a unified way.